main.cpp

```diff
@@ -122,394 +172,476 @@ struct CarWash
     PowerUnit PSU0;
-    //volt controlled attenuator
     VCA VCA0;
-    //left channle amplifier
     HeadphoneAmp AmpLeft;
-    //right channle amplifier
     HeadphoneAmp AmpRight;
-    //body of controller
     Body body0;
 
-    //control volume
     void setVol(int amount = 0);
-    //set input sources
     bool selectSource(int sourceNum, bool status=0);
-    //toggle crosfeed effect
     bool toggleCrossfeed(bool status = 0);
 };
-   
+
+void CanPlant::OfficeWorker::makeCall(int customerId, float time)
+{
+    SipProvider sipnet;
+    sipnet.makeCall(intNumber, customerId);
+    time++;
+}
+
+void CanPlant::OfficeWorker::pickCall(int number, float time)
+{
+    //int recordedNumber = number;
+    number = 0;
+    time++;
+}
+
+int CanPlant::OfficeWorker::reserveGood(int customerId, int amount)
+{
+    customerId++;
+    return amount++;
+}
+
+int CanPlant::produceCans(int fish, int tin)
+{
+    int amount = fish / tin;
+    return amount;
+}
+
+void CanPlant::loadBoxes(int boxes, int wareHouseNumber)
+{
+    boxes++;
+    wareHouseNumber++;
+}
+
+double CanPlant::reportScrapOut(double raw, double work)
+{
+    return raw * work *.01;
+}
+
+bool SipProvider::makeCall(int src, int dst)
+{
+    if (src == dst)
+    {
+        return false;
+    }
+    else
+    {
+        return true;
+    }
+}
+
+double SipProvider::chargeCustomer(double time, int customerId)
+{
+    if (customerId == 0)
+    {
+        return 0;
+    }
+    else
+    {
+        return time * price;
+    }
+}
+
+double SipProvider::traficCount(double time)
+{
+    return time * 0.1;
+}
+
+void Cat::eat(char foodType)
+{
+    if (foodType == 'F')
+    {
+        Cat::mew(3);
+    }
+}
+
+void Cat::sleep (float time)
+{
+    time--;
+}
+
+void Cat::mew (int count)
+{
+    count--;
+}
+
+void SpaceShip::CrewMember::examineAnimal(int date, float time, Cat cat)
+{
+    int testNum = 0;
+    if (cat.liveNumber <= 9)
+    {
+        recordTest(date, time, testNum++);
+    }
+    
+    date++;
+    time++;
+}
+
+bool SpaceShip::CrewMember::recordTest(int date, float time, int testNum)
+{
+    date++;
+    time++;
+    
+    if (testNum !=0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+void SpaceShip::CrewMember::examineCrew (int date, float time, CrewMember id)
+{
+    int testNum = 0;
+    if (id.weight >= 70.0f)
+    {
+        recordTest(date, time, testNum++);
+        date++;
+        time++;
+    }
+}
+
+bool SpaceShip::dock()
+{
+    SpaceShip Rassvet;
+    if (Rassvet.orbitHeight != 40.0f)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+void SpaceShip::makeLoop(int planetNum )
+{
+    planetNum++;
+}
+
+bool SpaceShip::takeOf(float startTime)
+{
+    SpaceShip Rassvet;
+    startTime--;
+    if (Rassvet.orbitHeight != 0.0f)
+    {
+        return true;
+    } 
+    else
+    {
+        return false;
+    }
+}
+
+void DAC::readInput(int channelNum)
+{
+    for (int i=0;i<channelNum;i++)
```

Comment:
make your code easier to read.
add spaces between your variables and the operators: `=, <, ;, etc`
- [ ] revise